ReSharper.FSharp/src/FSharp.Psi.Features/src/LanguageService/FSharpLanguageService.fs

```diff
@@ -24,7 +24,7 @@ open JetBrains.Util
 [<Language(typeof<FSharpLanguage>)>]
 type FSharpLanguageService
         (languageType, constantValueService, cacheProvider: FSharpCacheProvider, checkerService: FSharpCheckerService,
-         formatter: FSharpDummyCodeFormatter) =
+         formatter: IFSharpCodeFormatter) =
```

Comment:
What is the purpose of this interface? Is it going to be used in some way in future?